libraries/chain/webassembly/eos-vm-oc/LLVMJIT.cpp

```diff
@@ -133,7 +133,7 @@ namespace LLVMJIT
 			if(SectionName == ".stack_sizes") {
 				return stack_sizes.emplace_back(numBytes).data();
 			}
-			WAVM_ASSERT_THROW(isReadOnly);
+			//WAVM_ASSERT_THROW(isReadOnly);
```

Comment:
what's going on here?

Reply:
The wasmTable is not in a read-only section.

Reply:
ah okay. This is a safety check that llvm didn't generate a data section that it expects to write to. The table should be placed in a read only section, and the assert should go back in. AFAIK just setting the GlobalVariable const param to true and using private linkage works.

Reply:
I think I tried that, but eventually gave up.  In order to make it const, the initialization can't be handled externally.

Reply:
a35f580 has an implementation that works.